src/main/java/beeboo/command/AddCommand.java

```diff
@@ -0,0 +1,72 @@
+package beeboo.command;
+
+import beeboo.components.Storage;
+import beeboo.components.TaskList;
+import beeboo.components.Ui;
+import beeboo.exception.InvalidDateException;
+import beeboo.exception.NoDescriptionException;
+import beeboo.task.Deadlines;
+import beeboo.task.Events;
+import beeboo.task.ToDos;
+
+/**
+ * Represents a command to add a task to the chatbot's task list.
+ */
+public class AddCommand extends Command {
+    private String type;
+    private String command;
```

Comment:
For some of your subclasses of the Command class, you have a repeated attribute command. Perhaps consider making the attribute Protected, so that subclasses can access the inherited classes attributes.